internal/badges/service.go

```diff
@@ -0,0 +1,159 @@
+package badges
+
+import (
+	"context"
+	"math"
+
+	"github.com/google/go-github/v33/github"
+	"github.com/pkg/errors"
+)
+
+// Service is an interface for components that can handle requests for a badge.
+type Service interface {
+	// CheckBadge serves a badge based on check suite status.
+	CheckBadge(
+		ctx context.Context,
+		owner string,
+		repo string,
+		opts *CheckBadgeOptions,
+	) (CheckBadge, error)
+}
+
+type service struct {
+	// The following functions are usually provided by a GitHub client, but are
+	// overridable for testing purposes
+	getCombinedStatusFn func(
```

Comment:
I may have missed other references/uses of this method, but I don't seem to see usage in this file nor any instances of overriding in service_test.  Is it being used?

Reply:
This was associated with the other badge-building method that I removed. So this can go to. Will fix.

Reply:
Fixed